draft-ietf-quic-transport.md

```diff
@@ -612,8 +614,11 @@ The server includes a connection ID of its choice in the Source Connection ID
 field.  The client MUST use this connection ID in the Destination Connection ID
 of subsequent packets that it sends.
 
-The packet number field echoes the packet number field from the triggering
-client packet.
+The Packet Number field of a Retry packet MUST be set to 0.  This value is
+subsequently protected as normal. \[\[Editor's Note: This isn't ideal, because
+it creates a "cheat" where the client assumes a value.  That's a problem, so I'm
+tempted to suggest that this include any value less than 2^30 so that normal
+processing works - and can be properly exercised.]]
```

Comment:
This also makes a client that uses a 0-length CID vulnerable to injection of an arbitrary Retry packet.

Reply:
The 0-length connection ID is only possible after the client has received a Retry, after which it will discard any Retry packets that it receives.  So I don't think so.

Reply:
Right, of course.